clients/java/src/main/java/io/zeebe/client/api/command/CreateWorkflowInstanceCommandStep1.java

```diff
@@ -98,5 +99,14 @@
      *     it to the broker.
      */
     CreateWorkflowInstanceCommandStep3 variables(Object variables);
+
+    /**
+     * When this method is called, the response to the command will be received after the workflow
+     * is completed. The response consists of a set of variables.
+     *
+     * @return the builder for this command. Call {@link #send()} to complete the command and send
+     *     it to the broker
+     */
+    FinalCommandStep<WorkflowInstanceResult> withResult();
   }
 }
```

Comment:
```suggestion

}

interface CreateWorkflowInstanceWithResultCommandStep1 extends FinalCommandStep<WorkflowInstanceResult>
{
  // here we could add later more methods
}
```

Reply:
If you add this interface and return this on withResult() we are able to add more methods without breaking the public api.
It is also more consistent to the other interfaces.